fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/mixin/content/registries/MixinWorld.java

```diff
@@ -111,4 +124,22 @@ private void writeIdsToFile(BiMap<Integer, Identifier> idMap, File file) throws
 			fileReader.close();
 		}
 	}
+
+	@Unique
+	private void writeIdsToFile(File file, BiMap<Integer, String> idMap) throws IOException {
```

Comment:
read instead of write?

Reply:
That’s write. It uses strings instead of identifiers unlike the overloaded method.

Reply:
aren't you reading it from file

Reply:
yes
i'll change it to `read`
why wasn't the other one named as `read` too

Reply:
done